### PR TITLE
docs: update Groups documentation

### PR DESCRIPTION
**Pull request by furi0us333**
- Update screenshot as the link moved from menu to nav bar
- Add language for new group admin feature
- Update `user-menu.png`

### CODE REVIEW

File: docs/knowledge_base/groups.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ Phylum groups allow users to view/submit projects and analysis jobs in a shared
 2. [Create a project](../cli/commands/phylum_project_create.md) using the `--group` option and your group name
 3. [Analyze](../cli/commands/phylum_analyze.md) the desired dependency files
 
-Any user that is a member of the group will be able to access the analysis results.
+Any user that is a member of the group will be able to access the analysis results. Group administrators can add/promote members, demote administrators, and set group policy that applies to all projects in the group.
```
[the remark this answers is not in the document]
Comment (janasheehan) [reply docs/knowledge_base/groups.md:9]: for what it's worth, group admins can also ignore packages across the entire group

**Review by janasheehan — DISMISSED** [review]

File: docs/knowledge_base/groups.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (janasheehan) [reply docs/knowledge_base/groups.md:9]: added those comments for your consideration...

File: docs/knowledge_base/groups.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ Phylum groups allow users to view/submit projects and analysis jobs in a shared
...
 
-Group administration for adding/removing users, etc. can be found in the `Groups Admin` view on the [Phylum UI](https://app.phylum.io/auth/login).
+Group administration for adding/removing users, etc. can be found in the `Groups` view on the [Phylum UI](https://app.phylum.io/auth/login).
 
-![user menu](../../assets/user-menu.png)
```
Comment (maxrake) [inline docs/knowledge_base/groups.md:13]: This image is used in one other spot, for [the "API Keys" page](https://docs.phylum.io/knowledge_base/api-keys), so it is good that is was not deleted. It too is slightly off now with UI changes, but perhaps not so much that it matters for the context on that page.

**Review by maxrake — DISMISSED** [review]
One comment that _may_ result in changes, but approving now in case it doesn't.

File: docs/knowledge_base/groups.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (furi0us333) [reply docs/knowledge_base/groups.md]: updated, not diving into the ignore stuff at this time

File: docs/knowledge_base/groups.md (hunk 2 above)
Comment (furi0us333) [reply docs/knowledge_base/groups.md:13]: That is why I left it, but your guilt has prompted me to do the correct thing and update it 😝

**Review by maxrake — APPROVED** [review]
LGTM

**Pull request merged**